hdr_html_canvas_element.md

```diff
@@ -1,749 +1,336 @@
-standard conversion that is applied to all data in these situations, namely, a conversion using relative colorimetric intent.
-
-This operation is defined for [CSS Predefined Color Spaces](https://www.w3.org/TR/css-color-4/#predefined) in the HTML specification
-[here](https://www.w3.org/TR/css-color-4/#predefined-to-lab).
-
-In this section we define this conversion for the new predefined color spaces.
-
-These conversions are expressed using a connection color space with the system colorimetry specified in Rec. ITU-R BT.2100:
-
-* Red primary: `(0.708, 0.292)`
-* Green chromaticity: `(0.170, 0.797)`
-* Blue chromaticity: `(0.131, 0.046)`
-* White chromaticity (D65): `(0.3127, 0.3290)`
-
-_Note:_ The system colorimetry specified in Rec. ITU-R BT.2100 is identical to that specified in Rec. ITU-R BT.2020.
-
-The conversion from color space A to color space B is performed according to the following steps:
-
-* apply the inverse transfer function of color space A
-* convert to the connection space by multiplying by the connection matrix of color space A
-* convert to color space B by multiplying by by the inverse of the connection matrix of color space B
-* apply any linear light scaling to correctly map the level of perceived diffuse white level in to color space B
-* apply the transfer function of color space B
-
-The domain and range of this conversion consist of all real values and the component signals in the connection color space are real numbers.
-
-#### `srgb-linear`
-
-* Transfer function: identity
-
-* Connection matrix: matrix to convert the [sRGB primaries](https://www.w3.org/TR/css-color-4/#valdef-color-srgb) to the primary colours specified in Rec. ITU-R BT.2100
-
-* Linear light scaling: 1.0
-
-#### `srgb`
-
-* Transfer function: See [srgb-linear](#srgb-linear)
-
-* Connection matrix: matrix to convert the [sRGB primaries](https://www.w3.org/TR/css-color-4/#valdef-color-srgb) to the primary colours specified in Rec. ITU-R BT.2100
-
-* Linear light scaling: 1.0
-
-_Note:_ The domain of this transformation function is all real values. Its domain is not restricted to the unit interval [0, 1].
-
-Also see note in the Issues section at the bottom about whether this space should be distinct from the existing `'srgb'` space.
-
-#### `rec2100-hlg`
-
-
-* Transfer function: The HLG Reference inverse OETF specified at Rec. ITU-R BT.2100, scaled
-
-```javascript
-  function connectionTransferFunction(x) {
-    const a = 0.17883277;
-    const b = 1 - 4*a;
-    const c = 0.5 - a * Math.log(4 * a);
-    const k = (Math.exp((0.75 - c) / a) + b) / 12;
-    if (x < 0) {
-      return 0;
-    } else if (x <= 0.5) {
-      return x * x / (3 * k);
-    } else if (x <= 1) {
-      return (Math.exp((v - c) / a) + b) / (12 *  k);
-    } else {
-      return 1 / scale;
-    }
+```idl
+// SMPTE ST 2086 color volume metadata.
+dictionary CanvasSmpteSt2086Metadata {
```

Comment:
Does "SmpteSt2086" need to be part of the enum name? Is there some other metadata we plan to add in the future that needs its own dictionary?  I think calling this `CanvasMetadata` should suffice.